packages/api/src/get-build.js

```diff
@@ -58,9 +44,12 @@ class GetBuildHandler {
         const snapshot = await starterQuery.orderBy('timestamp', 'desc').limit(limit).get();
         const resp = [];
         snapshot.forEach(doc => resp.push(doc.data()));
-        res.send(resp);
+
+        var metadata = await this.client.collection(global.headCollection).doc(firebaseEncode(repoid)).get();
+
+        res.send({ metadata: metadata.data(), builds: resp });
       } catch (err) {
-        res.status(400).send({ error: err.message, stack: err.stack, name: err.name });
+        res.status(500).send({ error: err.message, stack: err.stack, name: err.name });
```

Comment:
Not a blocker right now,  but eventually we probably want to obfuscate the stack trace, unless we're in development mode.

You tend to cleanup errors before serializing them to the frontend, because they can contain sensitive information about how your system work.